build.gradle

```diff
@@ -77,12 +77,16 @@ tasks.withType(JavaCompile) {
 
 tasks.withType(Test) {
   useJUnitPlatform()
-
   testLogging {
     exceptionFormat = 'full'
   }
 }
 
+tasks.withType(Copy) {
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
```

Comment:
Any particular reason for introducing this strategy?  Not necessarily wrong just curious.

Reply:
I just thought it seemed reasonable and it fixed the build

https://docs.gradle.org/current/javadoc/org/gradle/api/file/DuplicatesStrategy.html <If you'd like me to choose a different one please let me know. But having duplicates of stuff around isn't always the best...